dev/tests/integration/testsuite/Magento/CatalogInventory/Model/StockItemSave/StockItemDataChecker.php

```diff
@@ -103,7 +115,10 @@ public function checkStockItemData($sku, array $expectedData)
         $productLoadedByModel = $this->productFactory->create();
         $productLoadedByModel->load($product->getId());
         $this->doCheckStockItemData($product, $expectedData);
-        $this->checkIntegrityWithInventory($product, $expectedData);
+        // soft dependency in tests because we don't have possibility replace tests from different modules
+        if ($this->moduleManager->isEnabled('Magento_Inventory')) {
```

Comment:
According to that MSI will be stored in separate repository
Need to revert all of this changes and copy all of tests to MSI current repository